### PR TITLE
[Processor] Fix rtmpdump not portable under Windows

### PR DESCRIPTION
**Pull request by cnbeining**
```
python3 you-get -p mpv http://www.zhanqi.tv/edmunddzhang
Site:       zhanqi.tv
Title:      【老E】正义的直播间
Type:       Flash video (video/x-flv)
Size:       inf MiB (inf Bytes)

Call rtmpdump:
rtmpdump -r rtmp://dlrtmp.cdn.zhanqi.tv/zqlive//9469_lNZYA?k=14e460f56dd93526eac8f92f0ec3d27a&t=57905434 -o - | mpv -

RTMPDump v2.4
(c) 2010 Andrej Stepanchuk, Howard Chu, The Flvstreamer Team; license: GPL
Connecting ...
INFO: Connected...
ERROR: RTMP_ReadPacket, failed to read RTMP packet header
```

So it should be working.

---
This change is [Reviewable](https://reviewable.io/reviews/soimort/you-get/1294)

### CODE REVIEW

**Comment by zhangn1985** [comment]
I don't get it..

**Comment by zhangn1985** [comment]
http://stackoverflow.com/questions/6338492/python-pipelining-subprocess-in-windows

I think this could be portable...

**Comment by zhangn1985** [comment]
https://www.microsoft.com/resources/documentation/windows/xp/all/proddocs/en-us/redirection.mspx?mfr=true

孤陋寡闻了，这个还真行。。。

File: src/you_get/processor/rtmpdump.py [hunk 1]
```diff
@@ -42,15 +42,31 @@ def download_rtmpdump_stream(url, title, ext,params={},output_dir='.'):
...
+    cmdline.append(url)
+    
+    #append other params if exist
     for key in params.keys():
-        cmdline+=key+" "+params[key] if params[key]!=None else ""+" "
-    cmdline+=" -o - | %s -"%player
-    print(cmdline)
-    os.system(cmdline)
+        cmdline.append(key)
+        if params[key]!=None:
```
Comment (soimort) [inline src/you_get/processor/rtmpdump.py]: Use `params[key] is not None`.

> Comparisons to singletons like None should always be done with is or is not , never the equality operators.

-- PEP-8

**Pull request merged**